README.md

```diff
@@ -46,3 +47,12 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+## My Response
+My complexity should be in the ball park of $\Theta(n^32^n)$ because I have to iterate through one set of nested for loops ( $\Theta(n^2)$ ) during which 
```

Comment:
Wait, where are the nested for loops? I only see nested loops where you set up the set of cities.